layers/+tools/pdf-tools/packages.el

```diff
@@ -0,0 +1,114 @@
+;;; packages.el --- pdf-tools Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: André Peric Tavares <andre.peric.tavares@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq pdf-tools-packages
+      '(
+        pdf-tools
+        ))
+
+(defun pdf-tools/init-pdf-tools ()
+  (use-package pdf-tools
+    :defer t
+    :mode (("\\.pdf$" . pdf-view-mode))
+    :config
+    (progn
+      (pdf-tools-install)
+      ;; TODO: Make `/', `?' and `n' work like in Evil
+      (evilified-state-evilify pdf-view-mode pdf-view-mode-map
+        ;; Navigation
+        "j"  'pdf-view-next-line-or-next-page
+        "k"  'pdf-view-previous-line-or-previous-page
+        "l"  'pdf-view-next-page-command
+        "h"  'pdf-view-previous-page-command
```

Comment:
In the current version, you'd get horizontal scrolling by using the arrows keys.

I tried to follow DocView (evilified) keybindings,  so `h` and `l` go to previous/next pages, which probably would be used more frequently than moving horizontally (but this is less consistent).

Let's wait and see if someone else suggests something...

DocView keybindings:
```
| ~h~         | previous page                            |
| ~H~         | adjust to height                         |
| ~j~         | next line                                |
| ~k~         | previous line                            |
| ~K~         | kill proc and buffer                     |
| ~l~         | next page                                |
| ~n~         | go to next search occurrence             |
| ~N~         | go to previous search occurrence         |
| ~P~         | fit page to window                       |
| ~r~         | revert                                   |
| ~W~         | adjust to width                          |
```